3.1/alpine3.14/amd64/Dockerfile

```diff
@@ -18,14 +18,12 @@ RUN wget -O vssnapshotdebugger.tar.gz "https://aka.ms/vssnapshotdebugger/release
 ENV CORECLR_ENABLE_PROFILING=1
 ENV CORECLR_PROFILER={324F817A-7420-4E6D-B3C1-143FBED6D855}
 ENV CORECLR_PROFILER_PATH_64=/diag/Instrumentation64/libInstrumentationEngine.so
+ENV MicrosoftInstrumentationEngine_ConfigPath64_SnapshotDebugger=/diag/Instrumentation64/ProductionBreakpoints_x64.config
 ENV MicrosoftInstrumentationEngine_FileLog=Errors
 ENV MicrosoftInstrumentationEngine_FileLogPath=/tmp/diag/log.txt
 ENV MicrosoftProductionDiagnostics_RuntimeFolder=/diag/Runtime
 ENV MicrosoftProductionDiagnostics_UtilsPath=/diag/Utils/ProductionBreakpointsUtils.dll
 
-# Install Visual Studio Debugger script prerequisites
-RUN apk add unzip
```

Comment:
Did you have a chance to validate that unzip is not required?

Reply:
Yes, unzip is not required. The vsdbg install script prefers to download tarball files on Linux starting with 16.5.